main/api/src/mill/api/Ctx.scala

```diff
@@ -82,8 +127,7 @@ class Ctx(
     with Ctx.Workspace {
 
   def dest: Path = dest0()
-  def length: Int = args.length
-  def apply[T](index: Int): T = {
```

Comment:
These weren't really used anywhere, and were just thin shims over `args` anyway, so I removed them